cli/backup/dashboard.go

```diff
@@ -162,9 +163,17 @@ func newListDashboardsCmd() simplecobra.Commander {
 
 			slog.Info("Listing dashboards for context", "context", config.Config().GetGDGConfig().GetContext())
 			for _, link := range boards {
-				url := fmt.Sprintf("%s%s", config.Config().GetDefaultGrafanaConfig().URL, link.URL)
+				base, err := url.Parse(config.Config().GetDefaultGrafanaConfig().URL)
+				var baseHost string
+				if err != nil {
+					baseHost = "http://unknown/"
```

Comment:
Should this be an error, or perhaps a warning instead of a placeholder string? Might be nice to make something pop out here, but maybe this is handling a case I'm not considering.

Reply:
This only comes into play after you've already connected to the service, retrieved all the data and returned the response.  A warning is fair, but an error is likely overkill as the data returned is still useful. It's just trying to be helpful to provide a direct URL for a given dashboard.